protostar/starknet/execution_state.py

```diff
@@ -10,12 +11,14 @@
 @dataclass
 class ExecutionState:
     starknet: ForkableStarknet
-    contract: StarknetContract
+    contract: Optional[StarknetContract]
```

Comment:
I would remove it from the global execution state and move it to the proper child-class if it is not the property of every ExecutionState

Reply:
Done